airbyte-connector-builder-server/connector_builder/impl/default_api.py

```diff
@@ -13,7 +13,56 @@
 
 class DefaultApiImpl(DefaultApi):
     async def get_manifest_template(self) -> str:
-        return "Hello World"
+        return """version: "0.1.0"
+
+        definitions:
+          selector:
+            extractor:
+              field_pointer: []
+          requester:
+            url_base: "https://example.com"
+            http_method: "GET"
+            authenticator:
+              type: BearerAuthenticator
+              api_token: "{{ config['api_key'] }}"
```

Comment:
As mentioned. In the generator this is defined as `/{{ config['api_key'] }}` because we need to escape the template so that the template generator doesn't fill this with anything. We want the `{{ }}` to actually be present in the file. But for the server version, we don't need to escape anything

Reply:
One way that you could reduce the duplication here would be to have a shared python method which takes in arguments for the api_token value and the spec title value, and returns this string with those arguments inserted. Then this method could call it and pass in the api_token without the slash, and the generator could call it and pass in the api_token with the slash, etc.

Reply:
hmm yeah that sounds like it could work, but I might table that type of change for the future, I don't think its necessarily critical to keep things perfectly in sync in V0. And we could have some thorny edges trying to read the YAML template from `airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs` across python packages 😅